common/build.gradle.kts

```diff
@@ -82,6 +81,10 @@ kotlin {
         implementation(deps.ktor.androidDriver)
     }
 
+    sourceSets["androidTest"].dependencies {
+        implementation("org.jetbrains.kotlin:kotlin-test-junit")
```

Comment:
Can this dependency be added to the dependencies in `buildSrc`?

Reply:
Sure! I'll put it under the Kotlin object 👍